lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -80,7 +89,7 @@ export class PublicClientApplication implements IPublicClientApplication {
 
     // Broker Objects
     private embeddedApp: BrokerClient;
-    private broker: BrokerManager;
+    private broker: boolean;
```

Comment:
Can we call this `isBroker` instead?

Reply:
Is this even needed since we have the config option `actAsBroker`?

Reply:
Good point, I did this temporarily.